src/features/ITGovernance/Admin/GRBReview/GRBReviewStatusCard/AddTimeOrEndVoting/index.tsx

```diff
@@ -0,0 +1,300 @@
+          }
+        );
+
+        resetModal();
+      })
+      .catch(() => {
+        setError(true);
+      });
+  });
+
+  return (
+    <>
+      {/* ADD TIME MODAL */}
+      <Modal
+        isOpen={isOpen === 'addTime'}
+        closeModal={() => resetModal()}
+        className="easi-body-normal padding-bottom-1 maxw-mobile-lg height-fit-content"
+      >
+        <h3 className="margin-top-0 margin-bottom-0">
+          {t('statusCard.addTimeModal.heading')}
+        </h3>
+
+        {err && (
+          <Alert type="error" slim>
+            {t('statusCard.addTimeModal.error')}
+          </Alert>
+        )}
+
+        <p>{t('statusCard.addTimeModal.description')}</p>
+
+        <RequiredFieldsText className="margin-top-0 font-body-sm" />
+
+        <Controller
+          control={control}
+          name="grbReviewAsyncEndDate"
+          render={({ field: { ref, ...field }, fieldState: { error } }) => (
+            <>
+              <FormGroup className="margin-top-0">
+                <Label htmlFor="grbReviewAsyncEndDate" requiredMarker>
+                  {t('statusCard.addTimeModal.label')}
+                </Label>
+
+                <p
+                  className="text-base-dark margin-y-1"
+                  style={{
+                    whiteSpace: 'pre-wrap'
+                  }}
+                >
+                  {t('statusCard.addTimeModal.hint')}
+                </p>
+
+                <ErrorMessage
+                  errors={errors}
+                  name="grbReviewAsyncEndDate"
+                  as={<FieldErrorMsg />}
+                />
+
+                <DatePickerFormatted
+                  id="grbReviewAsyncEndDate"
+                  {...field}
+                  dateInPastWarning
+                  value={field.value ?? ''}
+                  onChange={date => {
+                    if (date !== field.value) {
+                      field.onChange(date || ''); // Only update when there's a change
+                    }
+                  }}
+                />
+
+                {!!watch('grbReviewAsyncEndDate') && (
+                  <p className="text-base">
+                    {t('statusCard.addTimeModal.newEnd', {
+                      date: formatDateUtc(
+                        watch('grbReviewAsyncEndDate'),
+                        'MM/dd/yyyy'
+                      )
+                    })}
+                  </p>
+                )}
+              </FormGroup>
+            </>
+          )}
+        />
+
+        <ButtonGroup className="margin-top-4">
+          <Button
+            type="button"
+            data-testid="addTimeModalButton"
+            disabled={!watch('grbReviewAsyncEndDate')?.trim()}
+            onClick={() => {
+              addTimeOrEndEarly();
+            }}
+          >
+            {t('statusCard.addTimeModal.addTime')}
+          </Button>
+
+          <Button
+            type="button"
+            className="margin-left-2 margin-top-2"
```

Comment:
This back button also needs the `margin-top-2` class removed to be centered with the button to the left.

![Screenshot 2025-03-31 at 1 40 57 PM](https://github.com/user-attachments/assets/0aa751df-f16b-4567-97a8-e5b517080fa2)